packages/mobile/lib/auxiliary/scanner/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './actions'
+export * from './interfaces'
+export * from './stores'
```

Comment:
shouldnt we export also `helpers` here?

Reply:
No, helpers are supposed to be only for internal use in the same module.